MANIFEST.in

```diff
@@ -1,2 +1,2 @@
-include *.txt
-include *.ini

```

Comment:
I changed the line endings to UNIX. git will check them out as CRLF on windows if that's how people have it configured.